.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug report
+description: File a bug report to help us improve
+labels: ["bug", "needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ####
+        Before starting a bug report, please take a look into the [Troubleshooting Documentation](https://docs.edgeless.systems/contrast/troubleshooting) and revise the existing issue.
+        Further please ensure that the provided CLI logs are configured with the `--log-level` command-line flag set to `debug`.
+
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Contrast Version
+      description: "In which version do you encounter this bug? (Output of `contrast version`) "
+      placeholder: "v0.0.0"
+    validations:
+      required: true
+  - type: input
+    id: platform
+    attributes:
+      label: Deployment Platform
+      description: "On which platform do you encounter this bug? (Value of default_platform value in justfile.env)"
+      placeholder: "AKS-CLH-SNP"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Issue Description + Logs
+      description: A concise description of what you're experiencing. Please append the CLI logs(`--log-level debug`)  of the encountered issue right here.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce The Behavior
```

Comment:
Please revisit all labels: We follow the [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/welcome/) and only capitalize the first word in a headline.